lib/pdk/util.rb

```diff
@@ -1,8 +1,38 @@
 require 'tmpdir'
 require 'tempfile'
+require 'fiddle/import' if Gem.win_platform?
 
 module PDK
   module Util
+    module WindowsFunctions
+      if Gem.win_platform?
+        extend Fiddle::Importer
+        dlload 'kernel32.dll'
+        extern('int GetLongPathNameA(char*, char*, int)', :stdcall)
+
+        # Call into kernel32.dll to resolve short path e.g. C:\Users\~ADMIN1\foo
+        # should be changed to long path i.e. C:\Users\Administrator\foo
+        def long_path(short_path)
+          # Fetch long path name into a buffer
+          bufferlen = 255
+          buffer    = nil
+          loop do
+            buffer = ' ' * bufferlen
+            len    = GetLongPathNameA(short_path.to_s, buffer, buffer.size)
```

Comment:
There should be a check here if GetLongPathName fails (RC = 0)

https://msdn.microsoft.com/en-us/library/windows/desktop/aa364980(v=vs.85).aspx